mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDao.java

```diff
@@ -101,4 +101,11 @@ public interface JobExecutionDao {
    * @return collection of JobExecutionUserInfoCollection dtos with userIds, firstNames, lastNames
    */
   Future<JobExecutionUserInfoCollection> getRelatedUsersInfo(int offset, int limit, String tenantId);
+
+  /**
```

Comment:
Need to update java docs with some words what method doing

Reply:
Hi @SerhiiNosko - Updated code as per your suggestion. Thank you 🙂